src/dotnet/APIView/APIViewWeb/Client/css/shared/codeline.scss

```diff
@@ -355,23 +355,58 @@ code {
     color: #008509 !important;
 }
 
-.code-diagnostics-error {
-    background-color: var(--code-diagnostics-error-color);
-    border: 1px solid red;
+$diagnostics: ( 
+    fatal: var(--code-diagnostics-fatal-color), 
+    error: var(--code-diagnostics-error-color), 
+    warn: var(--code-diagnostics-warn-color), 
+    info: var(--code-diagnostics-info-color) );
+
+@each $type, $color in $diagnostics {
+    .code-diagnostics-#{$type} {
+        background-color: $color;
+    }
 }
 
-.code-diagnostics-warn {
-    background-color: var(--code-diagnostics-warn-color);
-    border: 1px solid #ffe066;
+.code-diagnostics p {
+    margin: 0;
 }
 
-.code-diagnostics-info {
-    background-color: var(--code-diagnostics-info-color);
-    border: 1px solid blue;
+.light-theme .code-diagnostics p a{
+    
+    &:active,
+    &:hover {
+        color: #00CBEB;
+    }
 }
 
-.code-diagnostics p {
-    margin: 0;
+.dark-theme .code-diagnostics p {
+    color: #e6eff7;
+
+    a {
+        color: #0d1dfc;
+
+        &:visited {
+            color: #3d4afd;
+        }
+
+        &:active,
+        &:hover {
+            color: #00CBEB;
+        }
+    }
+}
+
+.dark-solarized-theme .code-diagnostics a {
+    color: #0d1dfc;
+
+    &:visited {
+        color: #3d4afd;
+    }
+
+    &:active,
+    &:hover {
+        color: #00CBEB;
+    }
```

Comment:
Why not declare the colors as variables in `theme-colors.scss`? Better to have themes specification all in one place.